composer.json

```diff
@@ -17,9 +17,6 @@
         "symfony/console": "^2.7 || ^3.1",
         "symfony/options-resolver" : "^2.7 || ^3.1",
         "symfony/property-access": "^2.7 || ^3.1",
-        "symfony/asset": "^2.7 || ^3.1",
-        "symfony/templating": "^2.7 || ^3.1",
-        "symfony/translation": "^2.7 || ^3.1",
```

Comment:
those things are not needed at all, it was just added because the functional test enabled those components for no reason, see Tests/Functional/app/config/config.yml